sys/include/net/netopt.h

```diff
@@ -139,6 +139,17 @@ typedef enum {
     NETOPT_TX_END_IRQ,
     NETOPT_AUTOCCA,             /**< en/disable to check automatically
                                  *   before sending the channel is clear. */
+
+    /**
+     * @brief read-only check for a wired interface.
+     *
+     * If the interface is wireless this function will return -ENOTSUP, a
+     * positive value otherwise.
+     *
+     * @note Setting this option will always return -EONOTSUP.
+     */
+    NETOPT_IS_WIRED,
```

Comment:
Can you use the `netopt_enable_t` type here too? That `-ENOTSUP` is returned for `set` (or when the interface is not wired) is given if you document this option only for "getting". Look at the other flags to get an idea about how they are documented/use `netopt_enable_t`